### PR TITLE
Move websocket server to separate docker

### PR DESCRIPTION
**Pull request by nkoenig**
This moves the websocket server to a separate docker container in order to support scaling on AWS.

Signed-off-by: Nate Koenig <nate@openrobotics.org>

### CODE REVIEW

File: docker/cloudsim_websocket/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# so we need to workaround this using a force overwrite
+# Do this before installing ign-gazebo
+RUN /bin/sh -c 'echo "deb [trusted=yes] http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' \
+ && /bin/sh -c 'wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -' \
+ && /bin/sh -c 'apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654'
+
+# install ign-blueprint
+RUN apt-get update \
+&&  apt-get install -y \
+    ignition-blueprint \
```
Comment (AlejoAsd) [inline docker/cloudsim_websocket/Dockerfile:30]: Not sure this applies for the websocket server, but should this be using Ignition Dome?

File: docker/cloudsim_websocket/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# Commands below run as the developer user
+USER $USERNAME
+
+# When running a container start in the developer's home folder
+WORKDIR /home/$USERNAME
+
+# Copy entry point script, and set the entrypoint
+COPY cloudsim_websocket/run_websocket.bash ./
+COPY cloudsim_websocket/cloudsim_websocket.ign ./
```
Comment (AlejoAsd) [inline docker/cloudsim_websocket/Dockerfile:50]: I haven't tried building this, but if the Dockerfile is in the `docker/cloudsim_websocket` directory, and assuming you're running the `docker build` command from that directory, should these paths skip the `cloudsim_websocket` directory?

```suggestion
COPY ./run_websocket.bash ./
COPY ./cloudsim_websocket.ign ./
```

**Comment by peci1** [comment]
What about parametrizing the port number, too, with 9002 being the default? It would help running multiple simulations on one machine.

**Comment by nkoenig** [comment]
Not needed anymore.

**Pull request closed without merging**